testing_programs/generate_and_build.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 mkdir build/
 g++ -o build/demo_class -fdump-ipa-whole-program -flto -O0 demo_class.cpp demo_two.cpp
+g++ -static simple.cpp -o build/simple
```

Comment:
Is this due to it not being able to find functions in glibc which by default is dynamically loaded?